native-maven-plugin/src/functionalTest/groovy/org/graalvm/buildtools/maven/MetadataRepositoryFunctionalTest.groovy

```diff
@@ -87,17 +87,16 @@ class MetadataRepositoryFunctionalTest extends AbstractGraalVMMavenFunctionalTes
         outputContains NATIVE_IMAGE_EXE + " --exclude-config dummy/path/to/file.jar \"*\""
    }
 
-    void "if the path doesn't exist it throws an error"() {
+    void "if the path doesn't exist, the repository cannot be pulled"() {
         given:
         withSample("native-config-integration")
 
         when:
         mvn '-Pnative,metadataMissing', '-DquickBuild', '-DskipTests', 'package', 'exec:exec@native'
 
         then:
-        buildSucceeded
-        outputContains "GraalVM reachability metadata repository path does not exist"
-        outputContains "Reflection failed"
+        buildFailed
+        outputContains " Cannot pull GraalVM reachability metadata repository either from the one specified in the configuration or the default one"
```

Comment:
`" Cannot pull ...` (here) vs. `"Cannot pull ` in line 175.

Please create a constant that you use in both places.